src/con_duct/suite/main.py

```diff
@@ -87,6 +87,10 @@ def main(argv: Optional[List[str]] = None) -> None:
         help="Path to duct report files, only `info.json` would be considered. "
         "If not provided, the program will glob for files that match DUCT_OUTPUT_PREFIX.",
     )
+    parser_ls.add_argument(
+        "-e",
+        "--eval-filter",
```

Comment:
doc?  how user would know what fields are available ? (in particular due to flatten_dict use)